docs/guides/unreachable.rst

```diff
@@ -54,7 +54,7 @@ As an example, consider this simple calculator:
 .. code:: python
 
    import enum
-   from typing_extensions import Never
+   from typing import Never
```

Comment:
In general, are these "simple" replacements fine or should it be something more like 

```python
from typing import Never  # or from typing_extensions for Python < 3.11
```

?

Haven't seen any of the latter for other imports, but who knows...

Reply:
Let's discuss on the issue for now